src/documents/management/commands/document_importer.py

```diff
@@ -91,9 +91,15 @@ def _import_files_from_manifest(self):
                 continue
 
             doc_file = record["__exported_file_name__"]
+            thumb_file = record["__exported_thumbnail_name__"]
             document = Document.objects.get(pk=record["pk"])
             with open(doc_file, "rb") as unencrypted:
```

Comment:
nit: since we've now got two `with` blocks here, I'd appreciate it if you could add a new line around both as we have in a similar code block in the exporter.  It's just style really, but since you're here ... :-)